web/frontend/src/components/gamepad/index.svelte

```diff
@@ -167,6 +167,25 @@ const checkVal = (val?: number): number => {
   return val;
 };
 
+const trunc = (val?: number): number => {
+  const checkedVal = checkVal(val);
+  if (Number.isNaN(checkedVal)) {
+    return checkedVal;
+  }
+
+  // eslint-disable-next-line unicorn/no-unsafe-regex
+  const match = checkedVal.toString().match(/^-?\d+(?:.\d{0,4})?/u);
```

Comment:
Small request: could we comment the behavior of this regex? I always take way too long second guessing regex behavior 😂

Reply:
Done